cmd/kubelet/app/server.go

```diff
@@ -412,13 +429,222 @@ func run(s *options.KubeletServer, kcfg *KubeletConfig) (err error) {
 	return nil
 }
 
+// bootstrapClientCert will request a client cert for kubelet if '--bootstrapKubeconfig'
+// is non-empty and no client certificate is available in kubeconfig.
+// On success, the result certificate and key file will be stored on tmpfs (default to /var/run/kubernetes/),
+// and the kubeconfig will point to those files.
+// If '--bootstrapKubeconfig' is set but kubeconfig file doesn't exist, then the function will fail.
```

Comment:
So when `--kubeconfig` has `certpath`, which is empty file, and `--bootstrap-kubeconfig` also has `certpath`, which one should appear in the final kubeconfig? I plan to use what's in `--bootstrap-kubeconfig`, and if it's empty, then just fall back to default path (/var/run/kubernetes/) @liggitt